frontend/app/services/SubscriptionService.scala

```diff
@@ -155,6 +156,17 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
     } yield subscriptionDetails
   }
 
+  def getUsageCountWithinTerm(memberId: MemberId, unitOfMeasure: String): Future[Int] = (for {
+    account <- getAccount(memberId)
+    subscriptions <- zuoraRestService.subscriptionsByAccount(account.id)
```

Comment:
The same logic is applied to both clients of existing `subscriptionsByAccount` clients, and this is getting the last subscription. We can be DRYer at this point.